client/src/components/SidebarContainer.jsx

```diff
@@ -367,46 +359,20 @@ const SidebarContainer = (props: PropTypes) => {
           Criminal Laws
         </button>
       </div>
-      {tab === ARRESTS_TAB ? (
-        <div
-          id="Arrests"
-          class="tabcontent"
-          style={{ visibility: tab === ARRESTS_TAB ? "visible" : "hidden" }}
-        >
-          <SidebarChart arrests={chartData} laws={null} />
-        </div>
-      ) : null}
-
-      {tab === MASSAGE_PARLOR_LAWS_TAB ? (
-        <div
-          id="Massage Parlor Laws"
-          class="tabcontent"
-          style={{
-            visibility: tab === MASSAGE_PARLOR_LAWS_TAB ? "visible" : "hidden",
-          }}
-        >
-          <h3>Massage Parlor Laws</h3>
-        </div>
-      ) : null}
-      {tab === VACATUR_LAWS_TAB ? (
-        <div
-          id="Vacatur Laws"
-          class="tabcontent"
-          style={{
-            visibility: tab === VACATUR_LAWS_TAB ? "visible" : "hidden",
-          }}
-        >
-          <h3>Vacatur Laws</h3>
-        </div>
-      ) : null}
 
       {tab === ARRESTS_TAB ? (
```

Comment:
I know this isn't for this PR, but should we consider changing this conditional series into a switch?